libs/types/src/cdf/ballot-definition/convert.ts

```diff
@@ -217,9 +221,30 @@ export function convertVxfElectionToCdfBallotDefinition(
     );
   }
 
+  // In order to include contest term descriptions in the CDF, we need to
+  // create Office objects, since that's where Terms are defined.
+  const offices = vxfElection.contests
+    .filter(
+      (contest): contest is Vxf.CandidateContest =>
+        contest.type === 'candidate' && contest.termDescription !== undefined
+    )
+    .map(
+      (contest): Cdf.Office => ({
+        '@type': 'BallotDefinition.Office',
+        '@id': officeId(contest.id),
+        Name: text(contest.title), // Not used, but required
+        Term: {
+          '@type': 'BallotDefinition.Term',
+          Label: assertDefined(contest.termDescription),
+        },
+      })
+    );
+
```

Comment:
Might it be vaguely confusing to another (hypothetical) consumer of this data that we're only creating offices when there's a term description?

Reply:
That's fair. It seemed more confusing to me to have offices with no purpose. Without term they have no information. Do you think we should change it?

Reply:
Not really, just making sure it's intentional.